JSDemos/Demos/DataGrid/CustomSummaries/description.md

```diff
@@ -1 +1,15 @@
-The DataGrid support custom summary calculation. In&nbsp;this example, summaries are only calculated against selected grid rows. This behavior is&nbsp;implemented using the [calculateCustomSummary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/#calculateCustomSummary) and [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onSelectionChanged) properties.
\ No newline at end of file
+Custom summaries should be used if you want to implement your own logic. To add them on the client side, do the following:
```

Comment:
- Why "should"?
- "your own logic" of what?
- "To add them on the client side" - where else can I add them? Nothing have been said about server-side summaries yet.
- If I "add" custom summaries on the client side, does it mean they will be calculated on the client side?